packages/build-tools/src/steps/utils/expoUpdates.ts

```diff
@@ -29,15 +29,23 @@ export async function configureEASUpdateIfInstalledAsync({
   inputs: {
     runtimeVersion?: string;
     channel?: string;
+    throwIfNotConfigured: boolean;
```

Comment:
nit: I was thinking this "mode" could be based off a parameter provided to `configureEASUpdateIfInstalledFunction`. Then the `fn` could even check `getExpoUpdatesPackageVersionIfInstalledAsync` before calling (or not calling) `configureEASUpdateIfInstalledAsync`. This would keep the oddity of `throwIfNotConfigured` in the `BuildFunction`, not leaking it to utils.